src/backend/web/server.js

```diff
@@ -1,11 +1,30 @@
 require('../lib/config');
+const https = require('https');
+const fs = require('fs');
+
 const app = require('./app.js');
 
 const { logger } = app.get('logger');
 const HTTP_PORT = process.env.PORT || 3000;
+const HTTPS_PORT = process.env.HTTPS_PORT || 4443;
+
+const { key, cert } = (() => {
```

Comment:
If the HTTPS port isn't set, can we skip loading these?  If we leave it like this, it will break for every developer locally.